Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/HotWaterTemperatureReset.mo

```diff
@@ -124,10 +112,10 @@ First implementation.
           lineColor={0,0,127},
           textString="TOut"),
         Text(
-          visible=use_TRoo_in,
+          visible=TSetZon,
```

Comment:
This use of `visible` is incorrect and should be deleted. `visible` needs to be set to a boolean value. If `false`, the graphical element will be removed. Here, `visible` is no longer needed.